aws/resource_aws_route53_record_test.go

```diff
@@ -968,6 +1005,36 @@ resource "aws_elb" "main" {
 }
 `
 
+const testAccRoute53ElbAliasRecordUppercase = `
+resource "aws_route53_zone" "main" {
+  name = "notexample.com"
```

Comment:
I know other zones are static (so far), but can you randomize the name here, please? We can address others in a separate PR in the future.

Reply:
You got it

Reply:
Ha! You'll love this. We actually hardcode it in the exists/destroy functions too: https://github.com/terraform-providers/terraform-provider-aws/blob/b-aws_route53_record-alias-name-diffsuppressfunc/aws/resource_aws_route53_record_test.go#L586

Its all or nothing 👎  -- I will get that nitpick merged though 👍